# This will throw an error
try:
    int("a")
except ValueError:
    print("oops, you can't do that")

print("this is the end of my program")git 